getgather/mcp/patterns/hardcover-profile.html

```diff
@@ -1,7 +1,7 @@
 <html gg-domain="hardcover" gg-priority="3">
   <body>
     <div>
-      <button gg-autoclick gg-match="div#navigation a:has-text('Books')"></button>
+      <div gg-autoclick gg-match="//div[@id='navigation']//a[contains(text(), 'Books')]"></div>
```

Comment:
Not sure why the mcp's browser is not clicking anything when triggering it manually using the `button` element. But it works out with auto trigger using `div` element

Reply:
yes, it is.